internal/config/config_test.go

```diff
@@ -694,6 +697,7 @@ func TestLoad(t *testing.T) {
 			path: "./testdata/storage/git_provided.yml",
 			expected: func() *Config {
 				cfg := defaultConfig()
+				cfg.Experimental.FilesystemStorage.Enabled = true
```

Comment:
could we add some tests for when the experimental flag is set to `false` but the thing that is marked as `experimental` is set , and ensure that it is skipped when teh config is parsed?